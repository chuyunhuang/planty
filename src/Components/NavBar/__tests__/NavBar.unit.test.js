import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../NavBar';

test('render navbar', () => {
  const wrapper = shallow(
    <Navbar />
  );
  expect(wrapper).toMatchSnapshot();
})

