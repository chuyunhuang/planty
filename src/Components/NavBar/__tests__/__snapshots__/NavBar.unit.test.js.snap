// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render navbar 1`] = `
<nav
  className="navbar navbar-expand-sm navbar-dark px-sm-5"
  style={
    Object {
      "backgroundColor": "#DCF763",
    }
  }
>
  <Link
    to="/"
  >
    <img
      src="logo.png"
      style={
        Object {
          "backgroundColor": "#191919",
          "borderRadius": "50%",
          "width": "80px",
        }
      }
    />
  </Link>
  <ul
    className="navbar-nav align-items-center"
  >
    <li
      className="nav-item ml-5"
    >
      <Link
        className="nav-link"
        to="/"
      >
        Product
      </Link>
    </li>
  </ul>
  <Link
    className="ml-auto"
    to="/cart"
  >
    <button>
      Shopping Cart
    </button>
  </Link>
</nav>
`;
